config/config.js

```diff
@@ -4,20 +4,23 @@ const path = require('path');
 const rootPath = path.normalize(`${__dirname}/..`);
 
 module.exports = {
+  // application config
```

Comment:
Rather than adding comments, the object can be restructured to something like:
```js
{
  app: {
    name: ... ,
    root: ... ,
    port: ... ,
  },
  es : {
    ...
  },
  analytics: {
    ...
  },
```